config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -82,6 +84,8 @@ presubmits:
         command:
         - "./test-e2e/cip/e2e-entrypoint-from-container.sh"
         env:
+        - name: DOCKER_IN_DOCKER_ENABLED
```

Comment:
I think instead of this you're supposed to escalate privileges with

```
securityContext:
          privileged: true
```

Reply:
ack.